tests/test_virtualenv.py

```diff
@@ -337,6 +338,39 @@ def test_relative_symlink(tmpdir):
     assert os.path.exists(lib64)
 
 
+@pytest.mark.skipif(not hasattr(os, "symlink"), reason="requires working symlink implementation")
+def test_python_is_symlink(tmpdir):
+    """Test that the Python interpreter is copied in correctly if the entry
+    found in PATH is a symlink with a relative target, and a symlink to a
+    symlink (e.g. if Python was installed using stow or homebrew)."""
+
+    tmpdir = str(tmpdir)
+    orig_path = os.environ["PATH"]
+
+    # Set up bin/python -> bin/monty -> the real python, using relative links for both.
+    bin_path = os.path.join(tmpdir, "bin")
+    os.mkdir(bin_path)
+    real_python_path = distutils.spawn.find_executable("python")
+    symlink_1_path = os.path.join(bin_path, "monty")
+    os.symlink(os.path.relpath(real_python_path, bin_path), symlink_1_path)
+    symlink_2_path = os.path.join(bin_path, "python")
+    os.symlink("monty", symlink_2_path)
+
+    # Adjust PATH and check we find our new symlink.
+    os.environ["PATH"] = bin_path + ":" + orig_path
```

Comment:
let' use pytest ``monkeypatch`` here, modifying a global variable without resetting it is not nice

Reply:
you'll want to use here the ``os.pathsep``, as yes Windows does has symlink features